internal/cql/cql_parser.go

```diff
@@ -539,8 +539,6 @@ func (p *CQLParser) booleanValueExpression(_p int) (localctx IBooleanValueExpres
 	var _parentctx antlr.ParserRuleContext = p.GetParserRuleContext()
 	_parentState := p.GetState()
 	localctx = NewBooleanValueExpressionContext(p, p.GetParserRuleContext(), _parentState)
-	var _prevctx IBooleanValueExpressionContext = localctx
```

Comment:
This file is auto-generated, so it's problematic to change it.  Can it be removed from linting?

Reply:
Done!